### DISCUSSION TITLE
Adding Guest/Attendee list

### DISCUSSION THREAD

**Discussion by spinapplications** [add2cal/add-to-calendar-button#380] [Q&A] [answered]
Are there any plans to allow including a guest list in the calendar event?

**Comment by jekuer** [comment]
Hi @spinapplications,

What you can already do, is setting 1 attendee: https://add-to-calendar-button.com/configuration#attendee . This enables you to use the button as some kind of a dynamic invite.
At the moment, there is nothing else on the mid-term roadmap.

What would be your use case for it?
So far, we assumed that it makes not that much sense, because if people are on the "attendee list", why would they need another Add-to-Calendar-Button? They already got the invite.
It is different with 1, since there, you could argue that you want to dynamically send invite-like events. But with many, you would need to know them up-front and therefore, no case for the button. But maybe we missed something here.

**Reply by spinapplications** [reply]
Hello @jekuer,

Thank you for your quick reply. 

My use case is that if you are the host of an event that people could sign up for, then when you confirm the event you would add the event to the calendar including all the guests so that they would see it in their calendar also. 

I would appreciate it very much if this feature could be added, i.e. that you could include a list of attendees in the calendar event. Is there any chance that could happen?

**Reply by jekuer** [reply]
Please allow me one more question:
When you create the event and add those people as attendees, they would already receive an email (email is mandatory for an event attendee as per iCalendar specs) about the event. Why should they use the button to add the event a second time?

**Reply by spinapplications** [reply]
Let's say you have a website for a bridal shower and you ask people to add their names and email if they can attend on a certain date. Then, if you have the minimum amount of people who can attend you would confirm the date. Yes, you can simply email them, but if you, as the host, had a button on the website that added this event to your calendar it would be nice if you could add the guests as attendees as well so they'd have it in their calendar as well. Does that make sense?

**Reply by jekuer** [reply, marked as answer]
Thanks for the example!

**So, you want to inform about who else is attending without creating a real event in some calendar (because then they would automatically get the email invite, which has been my point before).**

I would say that it is better to not use the attendee functionality of the iCalendar format here, since this feature implicitly assumes that the event is hosted in some calendar, where people can (for example) also change their status by declining the event again. It would mislead the users.

What you could/should do in this case (as best practice), is rather adding the attendees as a list to the description.
There, you can even add them in a formatted way (like an unordered list (see https://add-to-calendar-button.com/advanced-use#case-2).
This also comes with the advantage that you can add the name only (without the email), which would be relevant for data privacy issues (unless you are not putting the button behind a login mask).
